daemons/gptp/common/ptp_message.cpp

```diff
@@ -1364,18 +1366,19 @@ void PTPMessagePathDelayRespFollowUp::processMessage(IEEE1588Port * port)
 	 * than a certain number of lost FUPs we should set asCapable
 	 * to false
 	 */
+	diffLastSeqId = sequenceId - port->getLastSeqId();
 	if( sequenceId != (port->getLastSeqId() + 1) )
```

Comment:
This will fix the rollover error here, because the next sequence number after `UINT16_MAX` should be 0 rather than `(int)(UINT16_MAX+1)`, which it will be in the code as written due to integer promotion.

    if( sequenceId != (uint16_t)(port->getLastSeqId() + 1) )